includes/data-stores/class-wc-admin-reports-customers-data-store.php

```diff
@@ -391,6 +391,34 @@ public function get_data( $query_args ) {
 		return $data;
 	}
 
+	/**
+	 * Gets the customer ID for given order, either from user_id, if the order was done
+	 * by a registered customer, or from the billing email in the provided WC_Order.
+	 *
+	 * @param WC_Order $order Order to get customer data from.
+	 * @return int|false The ID of the retrieved customer, or false when there is no such customer.
+	 */
+	public static function get_customer_id_from_order( $order ) {
```

Comment:
Originally, I created this function to have a possibility of getting the `customer_id` without creating it, if it's missing. 

However, thinking about it now in light of p90Yrv-XN-p2, I think we might be getting rid of `returning_customer` column, which means we probably don't need the `get_customer_id_from_order` function.

Reply:
@peterfabian Based on what I discovered in opening both PRs related to p90Yrv-XN-p2 we might be keeping `returning_customer` after all since my other method was less performant.

Let me know if you disagree with my assessment, but I think this PR is still a good one to get merged otherwise.